tool/fakerelay/main.go

```diff
@@ -42,6 +47,11 @@ func main() {
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		start := time.Now()
 		requestCount.Add(1)
+
+		if r.ContentLength > 0 {
+			bytesReceived.Add(r.ContentLength)
+		}
```

Comment:
```suggestion
		if r.ContentLength < 0 {
			panic("unexpected Content-Length")
		}
		bytesReceived.Add(r.ContentLength)
```

I'd avoid silently ignoring unhandled cases. Either make noise if we find something we didn't expect, or handle that case (for example by lazily reading the body like in [getsentry/sentry-go#168](https://togithub.com/getsentry/sentry-go/pull/168/files#diff-cdf1ba9d2aff2ab4ffd551b3b586b0220ffa76efe95e4926054f8abfcb50ed5cR116-R119) or more simply reading the body into a `bytes.Buffer` to get its length then set the buffer as the new body (to be possibly read again for `firstRequest`).